x-pack/plugins/monitoring/public/directives/logstash/pipeline_viewer/index.js

```diff
@@ -28,13 +30,14 @@ uiModule.directive('monitoringLogstashPipelineViewer', ($injector) => {
 
       scope.$watch('pipeline', (updatedPipeline) => {
         pipelineState.update(updatedPipeline);
-        const pipelineViewer = (
-          <PipelineViewer
-            pipelineState={pipelineState}
+        const configModel = Pipeline.fromPipelineGraph(pipelineState.config.graph);
+        const configViewer = (
+          <ConfigViewer
+            pipeline={List.fromPipeline(configModel)}
```

Comment:
I love how clean this changeover is! I imagine you will remove the files needed exclusively by `<PipelineViewer>` in a follow up PR?

Reply:
Totally nitpicky so feel free to ignore but you could make this prop like so to emphasize our conversion process a bit more clearly:

`pipeline={List.fromPipeline(Pipeline.fromPipelineGraph(pipelineState.config.graph`

Alternatively, you can leave your initialization of `configModel` above as-is but right below that add a line to initialize `listModel = List.fromPipeline(configModel)` and then pass the prop as `pipeline={listModel}`.

Either way I think by clubbing together the two conversion steps closer to each other, it might be just a tad bit clearer what's going on. WDYT?

Reply:
> I love how clean this changeover is! I imagine you will remove the files needed exclusively by <PipelineViewer> in a follow up PR?

Yep, it'll be the next thing I do when we've merged. Wanted to try to keep the PR clean.

> you could make this prop like so to emphasize our conversion process a bit more clearly

It's fine by me.. do you feel exploding it out like this makes it more or less readable than the single line conversion?

```
<ConfigViewer
  pipeline={
    List.fromPipeline(
      Pipeline.fromPipelineGraph(
        pipelineState.config.graph
      )
    )
  }
  timeseriesTooltipXValueFormatter={timeseriesTooltipXValueFormatter}
/>
```

Reply:
I like the exploded version better!

Reply:
Added in https://github.com/elastic/kibana/pull/18597/commits/96a29104a68ce859a0964f0a1d09c57be821e107